python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -778,6 +784,25 @@ def _computeReferenceOffsets(self, stdCat, bands):
         self.log.info("Found %d stars with at least 1 good observation in each band" %
                       (len(stdCat)))
 
+        # Associate each band with the appropriate physicalFilter and make
+        # filterLabels
+        filterLabels = []
+
+        lutPhysicalFilters = lutCat[0]['physicalFilters'].split(',')
+        lutStdPhysicalFilters = lutCat[0]['stdPhysicalFilters'].split(',')
+        physicalFilterMapBands = list(physicalFilterMap.values())
+        physicalFilterMapFilters = list(physicalFilterMap.keys())
+        for band in bands:
+            # Find a physical filter associated from the band by doing
+            # a reverse lookup on the physicalFilterMap dict
+            physicalFilterMapIndex = physicalFilterMapBands.index(band)
+            physicalFilter = physicalFilterMapFilters[physicalFilterMapIndex]
```

Comment:
It probably doesn't matter for the tests this is used in, but I suspect this will not give the results we want for e.g. HSC data that has both `HSC-I` and `HSC-I2`, because those are both `band="i"`.

Reply:
The way the lookup works is that we go from band to a physical filter and then to the standard filter.  In the case that we have both `HSC-I` and `HSC-I2`, these two physical filters will be mapped to the same standard filter (the default is `HSC-I2`).  This is enforced in the fgcm code that that you can't have a band mapped to more than one standard filter.  Therefore, we get `i` to `HSC-I` to `HSC-I2` *or* `i` to `HSC-I2` to `HSC-I2` but no matter the lookup path you end up with the same standard filter, and the one that was specified should be used (and thus get the correct color terms).  I'll add more comments in the code.